draft-ietf-anima-brski-prm.md

```diff
@@ -201,71 +198,58 @@ It may be at a central site or an internet resident "cloud" service.
 
 RER:
-: Registrar-enrollment-request is the CSR of a PER sent to the CA by the registrar (RA/LRA)
+: Registrar Enrollment-Request is the CSR of a PER sent to the CA by the domain registrar (RA/LRA).
 
 RVR:
-: Registrar-Voucher-Request is a request for a voucher signed by the Registrar to the MASA.
+: Registrar Voucher-Request is a request for a voucher signed by the domain registrar to the MASA.
 It may contain the PVR received from the pledge.
 
 This document includes many examples that would contain many long sequences of base64 encoded objects with no content directly comprehensible to a human reader.
 In order to keep those examples short, they use the token "base64encodedvalue==" as a placeholder for base64 data. 
 The full base64 data is included in the appendices of this document.
 
 This protocol unavoidably has a mix of both base64 encoded data (as is normal for many JSON encoded protocols), and also BASE64URL encoded data, as specified by JWS.
-The latter is indicated by a string like "BASE64URL(THING)"
+The latter is indicated by a string like "BASE64URL(content-name)".
+
 
 # Scope of Solution
 
 ## Supported Environments and Use Case Examples {#sup-env}
 
-BRSKI-PRM is applicable to environments where pledges are in pledge-responder-mode and may have no continuous connection to a domain registrar.
-Either way pledges are expected to be managed by the same registrar.
-
-This can be motivated by pledges deployed in environments not yet connected to the operational customer site/domain network, e.g., at a construction site where a building is going up.
-
-Another environment relates to the assembly of cabinets, which are prepared in advance to be installed on a customer site/domain.
-
-As there is no direct connection to the registrar available in these environments the solution specified allows the pledges to act in a server role so they can be triggered for bootstrapping e.g., by a commissioning tool.
-
-As BRSKI defines the pledge in a client role, initiating the bootstrapping (pledge-initiated-mode), BRSKI-PRM in contrast defines pledges acting as a server (pledge-responder-mode) responding to PVR and PER and consumption of the results.
-
-The following examples motivate support of BRSKI-PRM to support pledges acting as server as well as pledges with limited connectivity to the registrar.
-
-While BRSKI-PRM defines support for pledges in responder mode, there may be pledges which can act as both initiator or responder.
-In these cases BRSKI-PRM can be combined with BRSKI as defined in {{RFC8995}} or BRSKI-AE {{I-D.ietf-anima-brski-ae}} to allow for more bootstrapping flexibility.
+BRSKI-PRM is applicable to scenarios where pledges have no direct connection to the domain registrar, have no continuous connection, or require coordination of the pledge requests (e.g., to work in constrained environments).
+Either way, such pledges are expected to be managed by the same domain registrar as other BRSKI pledges.
```

Comment:
Great simplification! Yes, there was definitely an overlapping. The text in general fits. I will changed the first sentence proposed to:

_BRSKI-PRM is applicable to scenarios where pledges may have no direct connection to the domain registrar, may have no continuous connection, or require coordination of the pledge requests to be provided to a registrar._

I essentially included the "may" as the registrar-agent could be co-located with the registrar to just facilitate the communication with pledges in responder mode. so it ends up in a an additional functionality of the registrar. 

Regarding the part: _Either way, such pledges are expected to be managed by the same domain registrar as other BRSKI pledges._ I propose to delete the text as it may be misleading. Same registrar was meant to state that it is the one that takes care of the domain the pledge is bootstrapped for. Toereless had a similar comment and the intention was not to describe an operational model of a registrar. If the registrar is a distributed functionality should be independent from BRSKI-PRM. 

I propose to keep the following part as this statement regarding combined use was requested by one of the earlier reviews and simple underlines an option that the support of BRSKI-PRM does not exclude support of others. 
_While BRSKI-PRM defines support for pledges in responder mode, there may be pledges which can act as both initiator or responder.
In these cases BRSKI-PRM can be combined with BRSKI as defined in {{RFC8995}} or BRSKI-AE {{I-D.ietf-anima-brski-ae}} to allow for more bootstrapping flexibility._